charts/couchbase-community/README.md

```diff
@@ -0,0 +1,33 @@
+# ObservIQ Couchbase CE chart
+
+This chart deploys a StatefulSet with 3 replicas that are running Couchbase Community Edition.
+
+These nodes have basic configuration performed automatically on startup. The following actions are performed:
+
+* Set memory quotas
+* Configure services
+* Setup credentials
+* Enable memory-optimized indexes
+* Load samples
+
+## Creating a cluster
+
+The three replicas have basic configuration performed, but a cluster has not been created. To add and rebalance nodes, do the following:
+
+1. Forward port `8091` on the first pod in the statefulset by running the following command:
+```
+kubectl port-forward pod/couchbase-community-0 8091
+```
+
+2. Run the following command to get the IP addresses of the individual pods. Note the IP addresses for the second and third pods. 
+```
+kubectl get pod -o wide
+```
+
+3. Using your browser of choice, visit `localhost:8091` and you will see the login page for the node. The username and password will be `Administrator:password` for all nodes.
+
+4. After a successful login, go to the **Servers** page
```

Comment:
Before this happens, I had to configure a "new cluster"

![Screenshot from 2023-06-16 16-11-51](https://github.com/observIQ/charts/assets/23043836/868c6062-3c6a-4eee-b575-c0132fae5435)